src/main/api/ledger/binance/common.ts

```diff
@@ -0,0 +1,8 @@
+import { LedgerErrorId } from '../../../../shared/api/types'
+
+export const fromLedgerErrorType = (error: number): LedgerErrorId => {
```

Comment:
This is not currently used?

Reply:
That's correct, but we do need it as soon as we support Ledger Binance txs